applications/nrf5340_audio/src/audio/le_audio_rx.c

```diff
@@ -157,14 +159,23 @@ void le_audio_rx_data_handler(struct net_buf *audio_frame_rx, struct audio_metad
 
 	/* Check if we have received all frames, send if we have */
 check_send:
+	if ((rx_stats[channel_index].recv_cnt % 100) == 0 || (rx_stats[channel_index].recv_cnt % 100) == 1){
+			//LOG_WRN("ch index %d Encoded rx   L0 %d L1 %d", channel_index, audio_frame_rx->data[0], audio_frame_rx->data[1]);
+			//LOG_WRN("Encoded data L0 %d L1 %d  R0 %d R1 %d", audio_frame->data[0], audio_frame->data[1], audio_frame->data[(audio_frame->len / 2)], audio_frame->data[(audio_frame->len / 2)+1]);
+	}
+
 	if (le_audio_concurrent_sync_num_get() ==
 	    metadata_num_ch_get(net_buf_user_data(audio_frame))) {
 		/* We have received all frames we are waiting for, pass data on to
 		 * the next module
 		 */
+
+
+		
+		
 		ret = k_msgq_put(&ble_q_rx, (void *)&audio_frame, K_NO_WAIT);
 		ERR_CHK_MSG(ret, "Failed to put audio frame into queue");
-		audio_frame = NULL;
+		audio_frame = NULL; // This looks odd as the frame still needs to live for the queue reader to get it out.
```

Comment:
Alexander: is there a risk here of this impacting data extraction in the following module?